src/app/adminDashboard/teams/page.tsx

```diff
@@ -1,11 +1,85 @@
-import React from 'react'
+'use client'
+import React, { useState, useEffect } from 'react';
+import styles from '../../../styles/Dashboard.module.scss';
+import { Navbar } from '../../../Components/Navbar/Navbar';
+import TableComponent, { TeamsProfile } from "../../../Components/Table/TableData";
+//@ts-ignore
+import { Pagination } from 'technogetic-iron-smart-ui';
+import { BtnDashboard } from '../../../Components/CommonComponent/BtnDashboard';
+import sendRequest from '@/services/auth/auth_All_Api';
+import { useRouter, NextRouter } from 'next/router';
+
+export default function Teams() {
+    const [currentPage, setCurrentPage] = useState(1);
+    const [loading, setLoading] = useState<boolean>(false)
+    const [paginatedData, setPaginatedData] = useState<TeamsProfile[]>([]);
+    const rowPerPage = 8;
+    const totalPages = 10;
+
+    const fetchDataFromAPI = async () => {
+        setLoading(true)
+        try {
+            const response = await sendRequest('v1/addteam', {
+                method: 'POST',
+                headers: {
+                    'Content-Type': 'application/json',
+                },
+            });
+
+            if (!response) {
+                throw new Error('API response is not valid');
+            }
+
+            const data: TeamsProfile[] = response.data.data;
+            console.log("response", data);
+
+            setPaginatedData(data);
+        } catch (error) {
+            console.error('Error fetching data from API:', error);
+            setPaginatedData([]);
+        } finally {
+            setLoading(false)
+        }
+    };
+
+    useEffect(() => {
+        console.log("FUN")
+        fetchDataFromAPI();
+    }, []);
+
+    const onPageChange = (page: number) => {
+        setCurrentPage(page);
+    };
+
+
+    const columns: string[] = ['emails', 'leadPlayer', 'id'];
+
+
+    if (loading) {
+        return <div>Loading...</div>;
+    }
 
-function page() {
     return (
-        <div>
-            TEAMS
-        </div>
-    )
+        <>
+            <div className={styles.main_container}>
+                <div className={styles.abcd}>
+                    <div className={styles.sidebar_wrapper}>
+                        <Navbar />
+                        <BtnDashboard />
+                        {/* <TableComponent
+                            userData={paginatedData}
+                            columns={columns}
+                            showAdditionalButton={true}
+                        /> */}
+                        <Pagination
+                            currentPage={currentPage}
+                            totalPages={Math.ceil(columns.length / rowPerPage)}
```

Comment:
create a common function in utils globalFunction file and do it there gonna be use in many places